Makefile.am

```diff
@@ -13,10 +13,10 @@ library_includedir=$(includedir)
 
 library_include_HEADERS = inc/hclib-timer.h inc/hclib-rt.h inc/hclib.h \
 						  inc/hclib_config.h \
-						  inc/hclib-mak/hclib.mak inc/hclib-async.h src/inc/hclib-hpt.h \
-						  inc/hclib-forasync.h inc/hclib-promise.h inc/hclib_promise.h \
-						  src/inc/hclib-atomics.h inc/hclib-place.h inc/hcupc-support.h \
-						  inc/hclib-async-struct.h inc/hclib_cpp.h inc/hclib_future.h \
+						  inc/hclib-mak/hclib.mak inc/hclib-async.hpp src/inc/hclib-hpt.h \
```

Comment:
why is this unrelated change in a pull request about licensing? This looks like something that should be separated out into another pull request. This is just a general good practice, see #2 in http://blog.ploeh.dk/2015/01/15/10-tips-for-better-pull-requests/